css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 235px;
+  top: 20px;
```

Comment:
This is a _**big**_ movement. I would approach the movement from the side closest to where the element will be rather than the opposite side. In this case, that would be `top` and `right`